backend/src/server.ts

```diff
@@ -42,11 +42,13 @@ app.listen({ port: PORT, host: IP }, (err) => {
   // ca.crt is the default CA bundle provided by the service account for kubernetes
   // service-ca.crt is the CA bundle provided by the service account for kubernetes used by prometheus
   // odh-ca-bundle.crt and odh-trusted-ca-bundle.crt are the CA bundles provided by the ODH platform
+  // ca-bundle.crt is the default CA bundle used by the system
   const caPaths = [
     '/var/run/secrets/kubernetes.io/serviceaccount/ca.crt',
     '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt',
     '/etc/pki/tls/certs/odh-ca-bundle.crt',
     '/etc/pki/tls/certs/odh-trusted-ca-bundle.crt',
+    '/etc/pki/tls/certs/ca-bundle.crt',
```

Comment:
are you using CentOS or RHEL? If yes, then the default system CA bundle location is
`/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`

See
https://go.dev/src/crypto/x509/root_linux.go

@harshad16 at least in case of our standard notebook images, we always base default system trust on

https://github.com/opendatahub-io/kubeflow/pull/43/files#diff-1df80e242be90c94fce3ffb051a0bfe706fe924dab62438c7aa0a186b8067153R284

What do you think of this proposed change here?

Reply:
content of the configmap has two files still, right?
ca-bundle.crt:   being from central cluster config
odh-ca-bundle.crt:  being i.e. self-signed provided cert via DataScienceCluster

Reply:
Yes that's right @zdtsw we use ubi 8 as base image, symlink will point to the real path, which is the one you mentioned @shalberd, this is the standard we are going with alognside pipelines, cc @HumairAK

Reply:
is that symlink from `/etc/pki/tls/certs/ca-bundle.crt` to `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` there in all base images ... i.e. CentOS 9 Stream, UBI8?

When mounting in files via volume and volumeMount, the actual location has to be used, or can one mount to a symlink location? This more out of interest.

Reply:
This is a good discussion about which is the proper file reference as the default CA bundle.  I did a quick deep dive into CA files as the default location(s) and it appears that these locations are owned by the `ca-certificates` package on RPM based systems.

As @shalberd pointed out `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` is the current Red Hat default for generated certificates but `/etc/pki/tls/certs/ca-bundle.crt` is the historical default for Fedora/RHEL systems which is why the there is a symlink from `*/ca-bundle.crt --> */tls-ca-bundle.pem` in all of the generated images.

Either file reference should work as noted by the link to the `crypto/x509/root_linux.go` but if we are UBI8+ only we can safely use `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` as the default.  For any component that may need to support other distributions for workflows with custom user imagebuilds then it would be safer to rely on the `/etc/pki/tls/certs` as the universal default directory for openssl certificates

References:
* [Making CA certificates available to Linux command-line tools](https://www.redhat.com/sysadmin/ca-certificates-cli).
* cert-manager - [Managing public trust in kubernetes with trust-manager](https://cert-manager.io/docs/tutorials/getting-started-with-trust-manager/#configure-real-applications)

Reply:
> For any component that may need to support other distributions for workflows with custom user imagebuilds then it would be safer to rely on the /etc/pki/tls/certs as the universal default directory for openssl certificates

Yes, at least for Guillome's notebook images in odh-contrib, those are based on CentOS Stream 9.
I have been successful relying on the location `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` in those.
Meaning it is working with the current odh-notebook-controller and proxy implementation. Granted, of course, we have out Openshift cluster admins added own private CAs to the openshift-config.

Reply:
Ok, I've updated the PR, adding `'/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem'` as the default file and putting that at top of the toolchain. That's what I got from the conversation. Is that right? @LaVLaS @shalberd 
And thanks a lot for the great feedback here.